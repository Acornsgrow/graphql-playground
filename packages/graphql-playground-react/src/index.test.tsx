import * as React from 'react'
import { shallow } from 'enzyme'
import MiddlewareApp from './components/MiddlewareApp'

test('test middleware app', () => {
  const wrapper = shallow(
    <MiddlewareApp setTitle={true} showNewWorkspace={false} />,
  )
  expect(wrapper).toMatchSnapshot()
})
