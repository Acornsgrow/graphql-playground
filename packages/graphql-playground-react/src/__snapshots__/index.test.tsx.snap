// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test middleware app 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <Connect(PlaygroundWrapper)
    setTitle={true}
    showNewWorkspace={false}
  />
</Provider>
`;
